### DISCUSSION TITLE
Panel, repeat option, auto-resize

### DISCUSSION THREAD

**Discussion by DonatoD** [grafana/grafana#37097] [Dashboards, Panels & Visualizations]
I think it could be useful to have a panel option for height and width auto resize.
For example especially for tables where the number of showing rows is related to a variable. A feature even more useful when using the repeat panel option.
For example in my last case I have a dashboard completely based on a single panel and its repeat option. Being the panel height fixed, I can have a lot of empty space, as well as not enough space to show all the rows.
To solve this problem at the moment I need to create sever dashboard linked each other based on the number of rows that must be displayed, and all constituencies related to the other "standard" dashboard and their related links.
The auto resize would have been really useful and faster

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
